frontend/src/pages/Profile/UserCourses/CourseMaterialInfoContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Material } from 'types/course';
+
+import CourseMaterialInfo from './CourseMaterialInfo';
+
+interface MaterialProps {
+  material: Material;
+}
+
+const CourseMaterialInfoContainer: React.FC<MaterialProps> = ({ material }) => {
+  let materialTitle = '';
+  material.technology.forEach((technology, id) => {
+    if (id < material.technology.length - 1) {
+      materialTitle += `${technology}, `;
+      return;
+    } else {
+      materialTitle += technology;
+      return;
+    }
+  });
```

Comment:
Try to replace this logic with something more readable, I'm not sure it will work with our server